.github/workflows/main-build.yml

```diff
@@ -11,6 +11,9 @@ on:
       - '**.md'
   workflow_dispatch:
 
+  pull_request:
```

Comment:
I think you can just remove this again, unless we want to execute this workflow on every PR.

Reply:
Ok will do